test/unit/service.spec.js

```diff
@@ -1925,6 +1925,8 @@ describe("Test Service class", () => {
 			expect(Service.getVersionedFullName("posts")).toBe("posts");
 			expect(Service.getVersionedFullName("posts", 5)).toBe("v5.posts");
 			expect(Service.getVersionedFullName("posts", "testing")).toBe("testing.posts");
+			expect(Service.getVersionedFullName("posts", '5')).toBe("v5.posts");
+			expect(Service.getVersionedFullName("posts", '123abc')).toBe("123abc.posts");
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/moleculerjs/moleculer/pullRequest?prid=8141838)
```suggestion
			expect(Service.getVersionedFullName("posts", "123abc")).toBe("123abc.posts");
```